exception.py

```diff
@@ -0,0 +1,37 @@
+"""
+Define your custom class for exceptions. Show an example how it might be used.
+"""
+
+
+class Error(Exception):
+    pass
+
+
+class OtherTypeError(Error):
+    pass
+
+
+class DivideByZero(Error):
```

Comment:
Why didn't you rename this class?

Reply:
I thought he was in order, ready

Reply:
in which order?

Reply:
the name of the class, and at the end of the 'error', to make it clear that this is an exception